backoff/backoff.go

```diff
@@ -0,0 +1,71 @@
+package backoff
+
+import (
+	"time"
+)
+
+type RandomGenerator interface {
+	Int63n(n int64) int64
+}
+
+type Config struct {
+	Base                 time.Duration
+	Max                  time.Duration
+	BackOffResetDuration time.Duration
+	Jitter               Jitter
+}
+
+var DefaultConfig = &Config{
+	Base:                 time.Second,
+	Max:                  time.Second * 30,
+	BackOffResetDuration: time.Second * 60,
```

Comment:
I minor naming proposal. Let's not capitalize the `Off` part. I am not sure if there is a word `backoff` in English, but I am pretty sure that the technical people write it as one word. 

So it can be `BackoffResetDuration`.

Reply:
Makes sense yes, renamed it in the latest commit.